src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -355,6 +355,14 @@ export class CellModel extends Disposable implements ICellModel {
 		if (this.cellType !== CellTypes.Code) {
 			return;
 		}
+
+		// There can only be one tagged parameters cell in the Notebook
+		for (let cell of this.notebookModel.cells) {
```

Comment:
Could also just do 

`value = this.notebookModel.cells.find(cell => cell.isParameter) ? false : value;`

At the very least you should break out after setting value = false - no need to iterate all cells since you only care about finding the first.